doczrc.js

```diff
@@ -24,6 +24,8 @@ export default {
         'isRTL',
         'enableAutoPlay',
         'itemPadding',
+        'sidePadding',
+        'showEmptySlots',
```

Comment:
We are missing a `showEmptySlots.mdx` file with an example

Reply:
Yes, I forgot to add the file to my changes, like an idiot... Thank you, I'll fix that ASAP !

Reply:
(done)